code/controllers/subsystem/weeds.dm

```diff
@@ -56,6 +56,8 @@ SUBSYSTEM_DEF(weeds)
 			return
 		// Adds a bit of jitter to the spawning weeds.
 		addtimer(CALLBACK(src, .proc/create_weed, T, creating[T]), rand(1, 3 SECONDS))
+		if(prob(20))
+			addtimer(CALLBACK(src, .proc/create_node, T, creating[T]), rand(8, 10 SECONDS))
```

Comment:
This could lead to overloading the subsystem as it is.

We should look to ensure that new nodes that spawn are not within x distance of another node, and we're not hitting some upper limit.